### ISSUE TITLE
Doubt: About change at stacks-map.js

### ISSUE THREAD

**Issue by thadeu** [dougmoscrop/serverless-plugin-split-stacks#160]
If I adjust an existing `stacks-map.js` this way, I need recreating my stack?

My stacks-map.js looks like this

```js
module.exports = {
    'AWS::AppSync::ApiKey': { destination: 'AppSync', allowSuffix: true },
    'AWS::AppSync::DataSource': { destination: 'AppSync', allowSuffix: true },
    'AWS::AppSync::FunctionConfiguration': { destination: 'AppSync', allowSuffix: true },
    'AWS::AppSync::GraphQLApi': { destination: 'AppSync', allowSuffix: true },
    'AWS::AppSync::GraphQLSchema': { destination: 'AppSync', allowSuffix: true },
    'AWS::AppSync::Resolver': { destination: 'AppSync', allowSuffix: true },
    'AWS::CloudWatch::Alarm': { destination: 'Alarms', allowSuffix: true,  force: true}
 }
```

I may not recreate my stack! :(

**Comment by dougmoscrop** [comment]
New resources will follow stacks-map, existing resources stay where they are.

This is a limitation of Cloud Formation :(

There is support for import resources, https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-import.html

So, in theory this plugin could offer a command like `serverless split-stacks migrate --resource=ResourceName --target=StackName` but no work has been done on anything like that.  It would be fairly complex, you would need to:

- resolve any fn::imports that refer to that resources, substitute them with static hard coded values (e.g. ARN of something)
- set retention policy on the resource to keep
- remove the resources from the current stack and update the stack
- import the resource in to the new stack, exporting it like before, if that was the case
- update previous fn::imports that were temporarily hard-coded, to now import again

In other words, it would involve coordinating numerous, iterative updates to multiple stacks and nobody has tackled this AFAIK!
